src/manager.tsx

```diff
@@ -1,37 +1,220 @@
+      window.removeEventListener("offline", offline);
+      window.removeEventListener("online", online);
+    };
+  }, [setOffline]);
+
+  useEffect(() => {
+    if (localBuildProgress?.currentStep === lastStep.current) return;
+    lastStep.current = localBuildProgress?.currentStep;
+
+    if (localBuildProgress?.currentStep === "error") {
+      addNotification({
+        id: `${ADDON_ID}/build-error/${Date.now()}`,
+        content: {
+          headline: "Build error",
+          subHeadline: "Check the Storybook process on the command line for more details.",
+        },
+        icon: <FailedIcon color={color.negative} />,
+        onClick: clickNotification,
+      });
+    }
+
+    if (localBuildProgress?.currentStep === "limited") {
+      addNotification({
+        id: `${ADDON_ID}/build-limited/${Date.now()}`,
+        content: {
+          headline: "Build limited",
+          subHeadline:
+            "Your account has insufficient snapshots remaining to run this build. Visit your billing page to find out more.",
+        },
+        icon: <FailedIcon color={color.negative} />,
+        onClick: clickNotification,
+      });
+    }
+  }, [addNotification, clickNotification, localBuildProgress?.currentStep]);
+
+  const { startBuild, stopBuild } = useBuildEvents({
+    localBuildProgress,
+    accessToken,
+  });
+
+  let warning: string | undefined;
+  if (!projectId) warning = "Select a project";
+  if (!isLoggedIn) warning = "Login required";
+  if (gitInfoError) warning = "Git synchronization problem";
+  if (hasConfigProblem) warning = "Configuration problem";
+  if (isOffline) warning = "Not available while offline";
+
+  const clickWarning = useCallback(
+    () => openVisualTestsPanel(warning),
+    [openVisualTestsPanel, warning]
+  );
+
+  useChannel({
+    [TESTING_MODULE_RUN_ALL_REQUEST]: startBuild,
+    [TESTING_MODULE_CANCEL_TEST_RUN_REQUEST]: stopBuild,
+  });
+
+  if (warning) {
+    // eslint-disable-next-line jsx-a11y/anchor-is-valid
+    return <Link onClick={clickWarning}>{warning}</Link>;
+  }
+  if (running) {
+    if (localBuildProgress) {
+      const { renderProgress } = BUILD_STEP_CONFIG[localBuildProgress.currentStep];
+      return renderProgress(localBuildProgress);
+    }
+    return "Starting...";
+  }
+  if (isOutdated) {
+    return "Test results outdated";
+  }
+  if (localBuildProgress?.currentStep === "aborted") {
+    return "Aborted by user";
+  }
+  if (localBuildProgress?.currentStep === "complete") {
+    if (localBuildProgress.errorCount) {
+      return `Encountered ${pluralize("component error", localBuildProgress.errorCount, true)}`;
+    }
+    return changedStoryCount.length
+      ? `Found ${pluralize("story", changedStoryCount.length, true)} with ${pluralize(
+          "change",
+          changedStoryCount.length
+        )}`
+      : "No visual changes detected";
+  }
+  return "Not run";
+};
+
 addons.register(ADDON_ID, (api) => {
+  if (global.CONFIG_TYPE !== "DEVELOPMENT") {
```

Comment:
1. use globalThis instead
2. do you also want the panel to be disabled?
```suggestion
  if (globalThis.CONFIG_TYPE !== "DEVELOPMENT") {
```

Reply:
Yeah let's keep the panel, it has a special UI for static builds.